manifests/kustomize/third-party/argo/README.md

```diff
@@ -2,34 +2,33 @@
 
 Kubeflow Pipelines uses [Argo Workflows](https://argoproj.github.io/argo-workflows/) as the underlying workflow execution engine.
 
-This folder contains:
+This folder contains preconfigured Argo Workflows installations used in Kubeflow Pipelines distributions that use **remote references** to the upstream Argo Workflows repository instead of local copies.
 
-* `upstream/manifests` a mirror of argo workflows manifests upstream. It should never be edited here. Run `make update` to update it.
-* `installs` a folder with preconfigured argo workflows installations used in Kubeflow Pipelines distributions.
+Major differences from upstream argo manifests:
 
-  Major differences from upstream argo manifests:
+* Argo server is not included.
+* Argo workflow controller configmap is preconfigured to integrate with KFP.
+* Images are configured to use KFP redistributed ones which comply with open source licenses.
+* A default artifact repository config is added for in-cluster minio service.
 
-  * Argo server is not included.
-  * Argo workflow controller configmap is preconfigured to integrate with KFP.
-  * Images are configured to use KFP redistributed ones which comply with open source licenses.
-  * A default artifact repository config is added for in-cluster minio service.
+## Remote References Implementation
 
-## Upgrade argo
-
-Refer to [third_party/argo/README.md](../../../../third_party/argo/README.md).
+KFP uses remote Git references to Argo Workflows manifests instead of maintaining local copies. This approach:
 
-### Upgrade argo manifests
+* Eliminates licensing concerns about copying manifests
+* Simplifies deployment and upgrade processes
+* Ensures direct tracking of upstream changes
 
-Requirement: 
+All kustomization files reference manifests directly from the [Argo Workflows repository](https://github.com/argoproj/argo-workflows) using versioned Git references.
 
-Use kpt version above 1.0.0-beta.6, refer to [kpt installation](https://kpt.dev/installation/) for downloading kpt.
+## Upgrade argo
 
-As one step of above, we need to upgrade argo manifests in this folder.
+Refer to [third_party/argo/README.md](../../../../third_party/argo/README.md).
 
-1. Run:
+### Upgrade argo manifests
 
-    ```bash
-    make update
-    ```
+To upgrade to a new Argo version:
 
-    Note, argo version is pulled from [third_party/argo/VERSION](../../../../third_party/argo/VERSION). Edit the VERSION file first.
+1. Update the version in [third_party/argo/VERSION](../../../../third_party/argo/VERSION)
```

Comment:
Curious why all the `../` are needed
```suggestion
1. Update the version in [third_party/argo/VERSION](./VERSION)
```

Reply:
`VERSION` is in `third_party/argo/VERSION` while this README is in `manifests/kustomize/third-party/argo/README.md` (not the same path).